setup/templates/python-services/monitoring-refresh-config/monitoring-refresh-config.py

```diff
@@ -394,6 +397,10 @@ def check_monitoring_config_exists_and_writeable_or_quit():
 
 
 def main():
+    # Gov Cloud is in a different account, so we need to change region for SSM only
+    if 'us-gov' in fetch_ec2_region():
```

Comment:
It is not obvious to have the default region defined in one place and later override it with another region. I propose to implement `fetch_ec2_ssm_region` instead of `fetch_ec2_region` and use it  in the `Settings` class to init variable

Reply:
The function shouldn't be called ssm_region, since it pulls the region of the ec2 instance, not the needed ssm region.  I think you are right, though, so I rearranged and put the functions and included it in the class directly.  I tested, so I think it should work.